tests/options/no_color.rs

```diff
@@ -0,0 +1,15 @@
+use crate::common::TestDir;
+
+#[test]
+fn output_without_colors() {
+    let test_dir = TestDir::new();
+    let testfile_to_check = test_dir.create_testfile(".env", " BAR='Baz'\n");
+
+    let args = &["--no-color"];
+    let expected_output = format!(
+         "{a}:1 LeadingCharacter: Invalid leading character detected\n{a}:1 QuoteCharacter: The value has quote characters (\', \")\n\nFound 2 problems\n",
```

Comment:
To prepare an expected output use the [check_output](https://github.com/dotenv-linter/dotenv-linter/blob/master/tests/common/output.rs#L9) function.